//demostration of exception
import java.util.*;

public class pgm56_exception{
    public static void main(String args[]){
        Scanner in=new Scanner(System.in);
        int a,b;
        a=in.nextInt();
        b=in.nextInt();
        int result=a/b;
        System.out.println(result);
        System.out.println("rest of the code.......");

    }
    
}